core/examples/luigi-sample-angular/src/assets/extendedConfiguration.js

```diff
@@ -110,7 +110,20 @@ var projectDetailNavProviderFn = function(context) {
                     viewUrl:
                       '/sampleapp.html#/projects/' +
                       projectId +
-                      '/users/groups/:group/settings'
+                      '/users/groups/:group/settings',
+                    children: [
+                      {
```

Comment:
This doesn't look nice when clicked. I would recommend using the same approach as for the `groups` dynamic node

Reply:
I added `keepSelectedForChildren: true,` since we are not able to drill down here, because the child is dynamic. Think this is what you meant.